README.md

```diff
@@ -50,17 +50,17 @@ angular.module('myApp', ['angular-cron-gen'])
     $scope.cronExpression = '0 0 0/3 1/1 * ? *';
     $scope.isCronDisabled = false;
     $scope.cronOptions = {
-      formInputClass = 'form-control cron-gen-input', // Form input class override
-      formSelectClass = 'form-control cron-gen-select', // Select class override
-      formRadioClass = 'form-control-static cron-gen-radio', // Radio class override
-      formCheckboxClass = 'form-control-static cron-gen-checkbox', // Radio class override
-      hideMinutesTab = false, // Whether to hide the minutes tab
-      hideHourlyTab = false, // Whether to hide the hourly tab
-      hideDailyTab = false, // Whether to hide the daily tab
-      hideWeeklyTab = false, // Whether to hide the weekly tab
-      hideMonthlyTab = false, // Whether to hide the monthly tab
-      hideYearlyTab = false, // Whether to hide the yearly tab
-      hideAdvancedTab = true, // Whether to hide the advanced tab
+      formInputClass : 'form-control cron-gen-input', // Form input class override
```

Comment:
Thanks for catching these. Can you also remove the space between the key and the `:`?